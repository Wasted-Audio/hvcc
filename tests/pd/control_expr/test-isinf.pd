#N canvas 827 239 734 565 12;
#X obj 40 22 loadbang;
#X obj 40 49 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X msg 75 120 2;
#X obj 40 160 pow 1e+10;
#X msg 43 120 1;
#X obj 40 194 expr isinf($f1);
#X obj 40 80 t b b;
#X obj 40 230 print;
#X connect 0 0 1 0;
#X connect 1 0 6 0;
#X connect 2 0 3 0;
#X connect 3 0 5 0;
#X connect 4 0 3 0;
#X connect 5 0 7 0;
#X connect 6 0 4 0;
#X connect 6 1 2 0;
