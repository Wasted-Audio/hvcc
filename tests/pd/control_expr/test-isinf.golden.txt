[@ 0.000] print: 1
[@ 0.000] print: 0
